doc/computing/parallelism.rst

```diff
@@ -146,33 +157,23 @@ Note that:
   only use ``<LIB>_NUM_THREADS``. Joblib exposes a context manager for
   finer control over the number of threads in its workers (see joblib docs
   linked below).
-- Joblib is currently unable to avoid oversubscription in a
-  multi-threading context. It can only do so with the ``loky`` backend
-  (which spawns processes).
+- `threadpoolctl` internally manages the numbers of threads used by OpenMP
+  and BLAS implementations for scikit-learn implementations.
```

Comment:
The original item was about the lack of builtin mechanism to avoid oversubscription when using the threading backend of joblib.

```suggestion
- When joblib is configured to use the ``threading`` backend, there is no
  mechanism to avoid oversubscriptions when calling into parallel native
  libraries in the joblib-managed threads.
- All scikit-learn estimators that explicitly rely on OpenMP in their Cython code
  always use `threadpoolctl` internally to automatically adapt the numbers of
  threads used by OpenMP and potentially nested BLAS calls so as to avoid
  oversubscription.
```